src/internal/modules/gauging-stations/forms/remove-tag-confirm.js

```diff
@@ -0,0 +1,18 @@
+
+const Joi = require('joi');
+const { formFactory, fields } = require('shared/lib/forms/');
+
+const removeTagConfirmForm = request => {
+  const f = formFactory(request.path);
+
+  f.fields.push(fields.hidden('csrf_token', {}, request.view.csrfToken));
+  f.fields.push(fields.button(null, { label: 'Confirm' }));
+  return f;
+};
+
+const removeTagConfirmSchema = Joi.object().keys({
```

Comment:
You won't have know this at the time of writing but we have a helper to generate schema's with default options set (i.e. `abortEarly`). You can find it in `src/shared/lib/joi.helpers.js` and can be used as follows:

```
const removeTagConfirmSchema = createSchema({
   csrf_token: Joi.string().guid()
});
```

This will automatically add `Joi.object().keys()`, with global options. (P.S. rebase/merge `develop` to get this.